public/app/core/components/PageHeader/PageHeader.test.tsx

```diff
@@ -1,12 +1,10 @@
 import React from 'react';
 import PageHeader from './PageHeader';
-import { shallow, ShallowWrapper } from 'enzyme';
+import { render } from '@testing-library/react';
```

Comment:
```suggestion
import { render, screen } from '@testing-library/react';
```
Let's use `screen` to be consistent across tests.